src/chrome/content/rules/Inspectlet.com.xml

```diff
@@ -1,34 +1,32 @@
 <!--
-	Mismatched:
-		- apidocs.inspectlet.com
-
-	Mixed Content:
-		- blog.inspectlet.com
--->
-
-<!--
 
 	CDN buckets:
 		- d2ues8mtn1tc1i.cloudfront.net
-	Problematic subdomains:
-		- ^	(works; mismatched, CN: *.heroku.com)
 	inspectlet.js sets the following wildcard cookies
 	on whichever domain it is loaded from:
 		- __insp_nv
 		- __insp_pad
 		- __insp_ref
 -->
 
-<ruleset name="Inspectlet.com">
+<ruleset name="Inspectlet">
+
+	<!--	Mixed Content-->
+	<target host="blog.inspectlet.com" />
 
+	<rule from="^http://blog\.inspectlet\.com/wp\-content/(.*)" to="https://blog.inspectlet.com/wp-content/$1" />
```

Comment:
`(.*)` and `$1` is not needed here, that's what the core of HTTPS Everywhere already does.
You also don't need to escape `-`. But this won't work anyway since all other request will be redirected by the generic rule on [L28](https://github.com/EFForg/https-everywhere/pull/4575/commits/7f90d46fdbce55afe5d16f4a1cf39bb0451e55cf#diff-627e1fdb4727ad6e4139e127c7a09bc1R28)

But you could use a "negative exclusion" like this:
`<exclusion pattern="^http://blog\.inspectlet\.com/(?!favicon\.ico|wp-content/|wp-includes/)" />`
This requires three valid test urls that are not excluded and three that are excluded.